ecommerce/api_test.py

```diff
@@ -772,11 +777,14 @@ def test_enroll_user_in_order_items(mocker, user, has_redemption):
             == redemption.coupon_version.payment_version.company
         )
     created_course_run_enrollments = CourseRunEnrollment.objects.order_by("pk").all()
-    course_runs = [
-        run_enrollment.run for run_enrollment in created_course_run_enrollments
-    ]
+    course_runs = sorted(
+        [run_enrollment.run for run_enrollment in created_course_run_enrollments],
+        key=sort_func,
+    )
```

Comment:
Can we just try ordering the `CourseRunEnrollment` objects in the query? `CourseRunEnrollment.objects.order_by("run__pk").all()`. You wouldn't need the `sorted` calls, and the assertion at line 785 is redundant whether we make that change or not